openvpn/anonine/Sweden.ovpn

```diff
@@ -1,19 +1,26 @@
 client
-dev tun0
-proto udp 
+dev tun
+proto tcp
 ; Cert
-ns-cert-type server
-cipher BF-CBC
+remote-cert-tls server
+cipher AES-256-CBC
 ;Host
 resolv-retry infinite
 ;auth
-auth-user-pass /config/openvpn-credentials.txt
```

Comment:
This needs to be left as it is. It points to the file in the container that has the credentials